index/chat-iframe.njk

```diff
@@ -4,8 +4,9 @@
 {% block pageTitle %}Live chat - {{ super() }}{% endblock %}
 
 {% block content %}
+    {# https://eu.ccng.bt.com/ChatWeb/Default.aspx #}
     <iframe id="chat-iframe"
         title="Web Chat - Claim criminal injuries compensation - GOV.UK"
-        src="https://eu.ccng.bt.com/ChatWeb/Default.aspx">
+        src="https://services-ld.ccng.bt.com/ChatWeb/Default.aspx">
```

Comment:
Is this likely to change again? Would consider if it's worth parameterising it?

Reply:
this shouldn't need to change again